ui/StatusQ/src/StatusQ/Core/StatusScrollView.qml

```diff
@@ -49,6 +49,8 @@ Flickable {
     rightMargin: rightPadding
     contentWidth: contentItem.childrenRect.width
     contentHeight: contentItem.childrenRect.height
+    implicitWidth: contentWidth + leftPadding + rightPadding
+    implicitHeight: contentHeight + topPadding + bottomPadding
```

Comment:
Changes LGTM but we should double check with @alexandraB99 why they were removed in the first place

Reply:
But anyhow we need this in master cause all popups are broken (including all keycard flows) and remove this in a new PR, if it's really needed, adapting all necessary places to that change.